### ISSUE TITLE
Lemontree Plaza Error?

### ISSUE THREAD

**Issue by Huge-Succeeded** [DiscordDungeons/Bugs#608]
**Please describe your issue below.**  

Not knowing where one could and could not perform sides in the Primeval Kingdom, I attempted one side activity each in Buckleport and Ashgate, (mine, then fish, respectively, both of which returned the standard "Cannot do x here") followed by two in Lemontree Plaza. (Forage, which was successful, then chop, which returned "Cannot do x here," followed by the error produced below)

```js

DiscordAPIError: Missing Permissions
    at item.request.gen.end (/home/mackan/RPGBot/node_modules/discord.js/src/client/rest/RequestHandlers/Sequential.js:71:65)
    at then (/home/mackan/RPGBot/node_modules/snekfetch/src/index.js:218:21)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)

```

**Comment by Sven65** [comment]
That's strange, there's nothing in `#!chop` that needs any special permissions.

Are you sure it was caused by `#!chop`?

**Comment by Huge-Succeeded** [comment]
Well, I can't say 100% that it was caused by the chop command specifically, but the error was produced immediately after I used the chop command. It's possible one of the preceding commands I invoked may have caused the bug, and it just took a bit to generate the error due to being in a server detached from the main DRPG server? 

I attached a screenshot with as much of the preceding information as I could fit onto the screen, I'm hoping it helps. I humbly apologize that I didn't upload this with the original bug report, I was honestly highly perplexed on how uploading images to Github worked and didn't want to risk messing anything up. 

![drpgbugreport13-3-18](https://user-images.githubusercontent.com/37341333/37406091-b8efca82-276c-11e8-833b-cc1168af6c59.png)
